target/linux/ramips/image/mt76x8.mk

```diff
@@ -2,6 +2,10 @@
 # MT76x8 Profiles
 #
 
+define Build/zyimage
```

Comment:
Please move this define to target/linux/ramips/image/Makefile and drop it from target/linux/ramips/image/mt7620.mk as well.

Reply:
I made this in separate commit.